dffml/high_level/ml.py

```diff
@@ -140,6 +159,17 @@ async def accuracy(
             # through something like pydantic. See issue #36
             raise TypeError(f"{accuracy_scorer} is not an AccuracyScorer")
         # Run accuracy method
+        # if accuracy scorer expects predict feature(s), pass as argument else don't
+        if (
+            accuracy_scorer.__class__.__qualname__
+            not in NO_PREDICT_FEATURE_SCORERS
+        ):
+            if "features" in locals():
```

Comment:
Instead of checking if features is in locals, it is more Pythonic to set features to None, and then check `if features is not None`

Reply:
But let's remove this, we should always be passing features via expansion.

```python
return await actx.score(mctx, sctx, *features)
```